include/zephyr/sys/atomic.h

```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright (c) 1997-2015, Wind River Systems, Inc.
  * Copyright (c) 2021 Intel Corporation
- *
+ * Copyright (c) 2023 Nordic Semiconductor ASA
  * SPDX-License-Identifier: Apache-2.0
```

Comment:
Is there a reason you're changing the formatting here by removing the empty line between the last Copyright statement and the SPDX line? I did a quick grep through the Zephyr tree, and it seems having an empty line in between these is by far the most common way to format this.

Reply:
It takes less space. Are you nak-ing the PR over this?

Reply:
Fixed